packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -30,20 +30,16 @@ class CodeSizeController {
 
   void clearSnapshot() {
     _snapshotRoot.value = null;
-    clearSnapshotFile();
+    _snapshotRoot.value = null;
```

Comment:
this shouldn't be called twice right?

Reply:
Yup. Should be `_snapshotFile.value = null`